spec/features/user_can_log_in_spec.rb

```diff
@@ -0,0 +1,23 @@
+feature 'User can log in' do
+  context 'on log in form' do
+    before do
+      visit user_session_path
+      click_on 'Log in'
+    end
+  
+    it 'displays the log in form' do
+    expect(page).to have_content 'Log in'
+    expect(page).to have_field 'Email'
+    expect(page).to have_field 'Password'
+    end
+
+    it 'display options for sign up and forget password' do
+      expect(page).to have_link 'Sign up', href: '/users/sign_up'
+      expect(page).to have_link 'Forgot your password?', href: '/users/password/new'
+    end
+
+    it 'displays error message if no/wrong information completed' do
```

Comment:
please change this test so that you actually fill in the wrong password or email to get the error message. Use the factory bot to create a user and also write a test to successfully log that user in

Reply:
Okay. Noted. Thank you.